src/commands/profile.ts

```diff
@@ -0,0 +1,34 @@
+import { ApplicationCommandOptionType } from 'discord.js';
+
+export const profile: GlobalCommand = {
+	name: 'profile',
+	description: 'Responds with the Profile Picture of the mentioned user',
+	options: [
+		{
+			name: 'user',
+			description: 'The user to get the profile picture of',
+			type: ApplicationCommandOptionType?.User,
```

Comment:
`ApplicationCommandOptionType` isn't nullish, so we don't need to coalesce here.

```suggestion
			type: ApplicationCommandOptionType.User,
```

Reply:
The reason this failed the unit tests before is that we mock `discord.js` in `main.test.ts`. There are some ways to fix that:
- Import the actual `ApplicationCommandOptionType` object from `discord.js` and re-export that in the mock.
- Mock command/index.ts in `main.test.ts`. This may not work, depending on how deep that import is in the dependency tree.